<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kuttans</class>
 <widget class="QMainWindow" name="Kuttans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="Malayalam Unicode&lt;-&gt;ASCII Converter">Kuttans</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExportPDF"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Convert">
    <property name="title">
     <string>&amp;Convert</string>
    </property>
    <widget class="QMenu" name="menu_Unicode_To_ASCII">
     <property name="title">
      <string>&amp;Unicode To ASCII</string>
     </property>
     <addaction name="actionRevathi_u2a"/>
     <addaction name="actionIndulekha_u2a"/>
     <addaction name="actionKarthika_u2a"/>
     <addaction name="actionSelectFontMap_u2a"/>
    </widget>
    <widget class="QMenu" name="menu_ASCII_To_Unicode">
     <property name="title">
      <string>&amp;ASCII to Unicode</string>
     </property>
     <addaction name="actionRevathi_a2u"/>
     <addaction name="actionIndulekha_a2u"/>
     <addaction name="actionKarthika_a2u"/>
     <addaction name="actionSelectFontMap_a2u"/>
    </widget>
    <addaction name="menu_ASCII_To_Unicode"/>
    <addaction name="menu_Unicode_To_ASCII"/>
    <addaction name="separator"/>
    <addaction name="actionASCIIToUnicodePDF"/>
    <addaction name="actionUnicodeToASCIIPDF"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionKuttansHelp"/>
    <addaction name="actionPayyansHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Convert"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionExportPDF"/>
   <addaction name="actionPrint"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Edit Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/document-open.png</normaloff>:/Kuttans/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/window-close.png</normaloff>:/Kuttans/window-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/document-save.png</normaloff>:/Kuttans/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/document-save-as.png</normaloff>:/Kuttans/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionKuttansHelp">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/system-help.png</normaloff>:/Kuttans/system-help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Kuttans Help</string>
   </property>
  </action>
  <action name="actionPayyansHelp">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/system-help.png</normaloff>:/Kuttans/system-help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Payyans Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/face-confused.png</normaloff>:/Kuttans/face-confused.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionASCIIToUnicode">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;ASCII To Unicode</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/edit-copy.png</normaloff>:/Kuttans/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/edit-cut.png</normaloff>:/Kuttans/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/edit-paste.png</normaloff>:/Kuttans/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/edit-undo.png</normaloff>:/Kuttans/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/edit-redo.png</normaloff>:/Kuttans/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/edit-select-all.png</normaloff>:/Kuttans/edit-select-all.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Select All</string>
   </property>
  </action>
  <action name="actionSelectFontMap_u2a">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Custom Font map</string>
   </property>
  </action>
  <action name="actionSelectFontMap_a2u">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Custom Font map</string>
   </property>
  </action>
  <action name="actionRevathi_u2a">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Revathi</string>
   </property>
  </action>
  <action name="actionIndulekha_u2a">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Indulekha</string>
   </property>
  </action>
  <action name="actionKarthika_u2a">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Karthika</string>
   </property>
  </action>
  <action name="actionRevathi_a2u">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Revathi</string>
   </property>
  </action>
  <action name="actionIndulekha_a2u">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Indulekha</string>
   </property>
  </action>
  <action name="actionKarthika_a2u">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/preferences-desktop-font.png</normaloff>:/Kuttans/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Karthika</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/document-print.png</normaloff>:/Kuttans/document-print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="actionExportPDF">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/application-illustrator.png</normaloff>:/Kuttans/application-illustrator.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export As PDF</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/document-print-preview.png</normaloff>:/Kuttans/document-print-preview.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;rint Preview</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/edit-delete.png</normaloff>:/Kuttans/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/document-new.png</normaloff>:/Kuttans/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionASCIIToUnicodePDF">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/application-illustrator.png</normaloff>:/Kuttans/application-illustrator.png</iconset>
   </property>
   <property name="text">
    <string>ASCII &amp;PDF to Unicode PDF</string>
   </property>
  </action>
  <action name="actionUnicodeToASCIIPDF">
   <property name="icon">
    <iconset resource="../qrc/kuttans.qrc">
     <normaloff>:/Kuttans/application-illustrator.png</normaloff>:/Kuttans/application-illustrator.png</iconset>
   </property>
   <property name="text">
    <string>Unicode P&amp;DF to ASCII PDF</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/kuttans.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>activated(int)</signal>
   <receiver>Kuttans</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>activated(int)</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>activated(int)</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>activated(int)</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>activated(int)</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>activated(int)</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectAll</sender>
   <signal>activated(int)</signal>
   <receiver>textEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
